railties/test/application/middleware/cookies_test.rb

```diff
@@ -103,7 +103,7 @@ def read_raw_cookie
 
       require "#{app_path}/config/environment"
 
-      verifier_sha512 = ActiveSupport::MessageVerifier.new(app.key_generator.generate_key("sha512 salt"), digest: :SHA512)
+      verifier_sha512 = ActiveSupport::MessageVerifier.new(app.key_generator.generate_key("sha512 salt"), digest: "SHA512")
```

Comment:
Why you needed this change? If it because symbols are not valid anymore we need to fix this.

Reply:
Yes, it will break but it's an easy change that as I described here: https://github.com/rails/rails/pull/42919/files#r700314987

I'll do so. Thanks for the feedback!